website/docs/docs/deploy/ci-jobs.md

```diff
@@ -77,6 +76,15 @@ If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cl
       - `non_native_pull_request_id` (for example, BitBucket)
    - Provide the `git_sha` or `git_branch` to target the correct commit or branch to run the job against. 
 
+## Test metrics in CI <Lifecycle status="team,enterprise" />
+Automatically test your metric definition models (metrics, semantic models, and saved queries) during code reviews by running warehouse validation checks in your CI job.
```

Comment:
>Automatically test your metric definition models (metrics, semantic models, and saved queries) during code reviews by running warehouse validation checks in your CI job.

I've been referring to ` metric definition models `as semantic nodes, i.e the nodes in your dag related to semantics. Does that jive with you? I would update this sentence to:

>Automatically test your semantic nodes (metrics, semantic models, and saved queries) during code reviews by running warehouse validation checks in your CI job.

Reply:
perfect, updated all!